src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -1303,7 +1306,7 @@ void runtime::cpu::CPU_ExternalFunction::register_common_passes(
     REGISTER_KNOBBED_PASS(CPUCollapseDims, true, runtime::cpu::pass)
 
 #ifdef NGRAPH_MLIR_ENABLE
-    if (std::getenv("NGRAPH_MLIR") != nullptr)
+    if (getenv_bool("NGRAPH_MLIR_CALLBACK"))
```

Comment:
This should be NGRAPH_MLIR instead of NGRAPH_MLIR_CALLBACK, made a mistake when doing copy and paste. These checks were here before. I think for line 1300 we don't need NGRAPH_MLIR_ENABLE because we run CPUFusion pass when 1) NGRAPH_MLIR_ENABLE is false or 2) when NGRAPH_MLIR_ENABLE is true but NGRAPH_MLIR is false.